lib/Doctrine/ORM/EntityManager.php

```diff
@@ -156,15 +156,15 @@ class EntityManager implements EntityManagerInterface
      * Creates a new EntityManager that operates on the given database connection
      * and uses the given Configuration and EventManager implementations.
      */
-    public function __construct(Connection $conn, Configuration $config)
+    public function __construct(Connection $conn, Configuration $config, ?EventManager $eventManager = null)
```

Comment:
The class-level phpdoc comment refers to the factory method your deprecated, please update it.